src/Zynga/Framework/Cache/V2/Config/InMemory/Base.hh

```diff
@@ -0,0 +1,22 @@
+<?hh // strict
+
+namespace Zynga\Framework\Cache\V2\Config\InMemory;
+
+use Zynga\Framework\Cache\V2\Config\Base as ConfigBase;
+
+abstract class Base extends ConfigBase {
+
+  public function getServerPairings(): Map<string, int> {
+    $hosts = Map {};
+    $hosts['127.0.0.1'] = 11211;
```

Comment:
Do you really need this if there are no hosts to connect to? I would say give some thought as to what all you need to get this to work.